scripts/files/geotiff.py

```diff
@@ -1,11 +1,9 @@
-from typing import Dict, List, Tuple
-
 from shapely.geometry import Polygon
 
 from scripts.gdal.gdalinfo import GdalInfo
 
 
-def get_extents(gdalinfo_result: GdalInfo) -> Tuple[Dict[str, List[float]], Tuple[float]]:
+def get_extents(gdalinfo_result: GdalInfo) -> tuple[dict[str, list[list[list[float]]]], tuple[float, float, float, float]]:
```

Comment:
how come this converts to `list` from `List` where as everything else is `List` ?

Reply:
[`List` is part of the deprecated type aliases from pre-Python 3.9](https://docs.python.org/3/library/typing.html#deprecated-aliases) - we can use the built-in types instead nowadays.